### DISCUSSION TITLE
[Projects Beta] Narrow down "Write" role's permissions

### DISCUSSION THREAD

**Discussion by redcatbear** [community/community#7713] [Projects and Issues]
As role "Administrator",
I want to change the settings and structure of a project,
in order to set up boards, tables and basic filters.

As role "Write",
I want to change contents of tickets in a view, like entering a due date or selecting a complexity for an issue,
in order to work with the issues.

As role "Write",
I don't want to be able to change project settings,
in order not to break the setup that the administrator made.

## See also

* #7587

**Comment by sreuterle** [comment]
I fully agree to that! I want to provide a project view to my team and dont' want them to be able to change the settings of the view itself. The suggested solutions seems to be a good option from my point of view too.
